libraries/redshop/tags/sections/attributes.php

```diff
@@ -650,16 +650,21 @@ function ($item) {
                 $property->property_price = $property->property_price_without_vat;
             }
 
-            $attributesPropertyOldPrice = $property->property_price;
-
             $pricelist = RedshopHelperProduct_Attribute::getPropertyPrice($property->value, 1, 'property');
 
             if (!empty($pricelist)) {
                 $property->property_price = $pricelist->product_price;
+                $attributesPropertyOldPrice = $pricelist->discount_price;
+            }
+            else{
+                $attributesPropertyOldPrice = $property->property_price;
```

Comment:
@hoangvan149dhv So with operand not `=` set this is totally wrong!!!
cc @NguyenBao10